inc/wc-calypso-bridge-hide-alerts.php

```diff
@@ -12,3 +12,14 @@
 // Hide Apple Pay and Google Payment notices
 add_filter( 'pre_option_wc_stripe_show_apple_pay_notice', '__return_true' );
 add_filter( 'pre_option_wc_stripe_show_request_api_notice', '__return_true' );
+
+// Hide setup store notice.
+add_filter( 'woocommerce_show_admin_notice', 'wc_calypso_bridge_hide_admin_notice' );
+
+function wc_calypso_bridge_hide_admin_notice( $notice ) {
```

Comment:
I think we need to capture [both arguments here ($bool, $notice)](https://github.com/woocommerce/woocommerce/blob/7eedf1c5134607c3d38b085fc88721fc5d2d14af/includes/admin/class-wc-admin-notices.php#L154) so that we can return the correct value, instead of always returning `true`.